config/webpack.lib.config.js

```diff
@@ -15,13 +15,15 @@ module.exports = {
     bundle: path.resolve(__dirname, '../src/lib-voyager.tsx'),
   },
   output: {
-    filename: "[name].js",
+    filename: "lib-voyager.js",
     path: path.resolve(__dirname, '../lib'),
     // Add /* filename */ comments to generated require()s in the output.
     pathinfo: true,
     // There are also additional JS chunk files if you use code splitting.
     chunkFilename: 'static/js/[name].chunk.js',
-    publicPath: '/lib/'
+    publicPath: '/lib/',
```

Comment:
Since other Vega libraries use `build/` as the compiled folder file, I wonder if we should be consistent. 

That said, other libraries are toolkit but Voyager is a UI component, so we don't really have to be the same.  
(Just asking)